scripts/vep/run_vep.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+
+"""
+Run VEP on the hail mt
+"""
+
+
+import click
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+
+@click.command()
+@click.option('--mt', required=True, help='Hail matrix table to run VEP on')
+def main(mt: str):
+    """
+    Run vep using main.py wrapper
+    """
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(mt)
+    # filter to biallelic loci only
+    mt = mt.filter_rows(hl.len(mt.alleles) == 2)
+    mt = mt.filter_rows(mt.alleles[1] != '*')
+    vep = hl.vep(mt, config='file:///vep_data/vep-gcloud.json')
+    vep = vep.rows()
+    vep_path = output_path('vep105_GRCh38.mt')
```

Comment:
Didn't we want to use VEP 104 for TOB-WGS?

Reply:
Is this ready to go now? If so I can change it

Reply:
Can you check with @vladsavelyev about this, please? He's actively working on that.